mix.exs

```diff
@@ -25,7 +35,34 @@ defmodule Mpdecimal.MixProject do
     [
       {:decimal, "~> 2.0"},
       {:elixir_make, "~> 0.6", runtime: false},
-      {:benchee, "~> 1.0", only: :dev},
+
+      {:ex_doc, "~> 0.24.2", only: :dev},
+      {:benchee, "~> 1.0", only: :dev}
+    ]
+  end
+
+  defp package do
+    [
+      description: "Provide fixed-decimal precision math as a NIF",
+      files: [
```

Comment:
normally we wouldn't have to specify this field, but because we are using some non-standard directories, we need to specify which directories and files we want to publish when we publish the package. I *think* I got all the relevant files here, but I would love a second set of eyes to make sure we aren't including something that shouldn't be publishing or including something we don't want published.